package.json

```diff
@@ -3,6 +3,7 @@
     "test": "lerna exec -- jest --coverage",
     "test:watch": "lerna exec -- jest --watch --coverage",
     "lint": "eslint ./packages/*/src/**/*.js",
+    "link": "lerna link",
     "bootstrap": "lerna exec --parallel -- npm i",
```

Comment:
I think we should change `lerna exec --parallel -- npm i` to [`lerna bootstrap`](https://github.com/lerna/lerna/tree/master/commands/bootstrap#readme) in package.json, so the config.yml can run `yarn bootstrap` instead of `yarn build` and `yarn link`

Reply:
Thanks! I've changed `lerna exec --parallel -- npm i` to `lerna bootstrap` in package.json but it looks like we still need `yarn build` or it would throw the syntax error.